cmd/cli/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+
+	"golang.org/x/crypto/ssh"
+)
+
+// remote PODMAN_HOST=ssh://<user>@<host>[:port]/run/podman/podman.sock
+// local  PODMAN_HOST=unix://run/podman/podman.sock
+
+var (
+	DefaultURL = "unix://root@localhost/run/podman/podman.sock"
```

Comment:
Shouldn't this be `ssh://` if it includes user & host?

Reply:
Golang parses this well, and it drives home this access is root by default